System.Compression.Primitives/System/IO/Compression/Brotli/BrotliNative.cs

```diff
@@ -0,0 +1,308 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+namespace System.IO.Compression
+{
+    /// <summary>
+    /// This class provides declaration for constants and PInvokes as well as some basic tools for exposing the
+    /// native Brotli library to managed code.
+    /// </summary>
+   internal static partial class BrotliNative
+    {
+        public enum BrotliDecoderResult : int
+        {
+            /// <summary>
```

Comment:
Nit, can you combine the summary comments and add it above the entire enum instead of above each individual enum value?
i.e. 
```C#
/// <summary>
/// 0 - Decoding error, e.g. corrupt input or memory allocation problem
/// 1 - Partially done; should be called again with more input
...
///</summary>
```